pkg/services/libraryelements/api.go

```diff
@@ -30,11 +30,31 @@ func (l *LibraryElementService) createHandler(c *models.ReqContext) response.Res
 		return response.Error(http.StatusBadRequest, "bad request data", err)
 	}
 
+	if cmd.FolderUID != nil {
```

Comment:
when folderUID is in the input, we take UID, when it is empty, it is general folder, otherwise retrieve folder with uid.